content/src/service/access/AccessCheckerImpl.ts

```diff
@@ -1,356 +1,37 @@
 import log4js from "log4js"
-import ms from "ms";
 import { EntityType, Pointer, Timestamp, Fetcher } from "dcl-catalyst-commons";
 import { AccessChecker } from "./AccessChecker";
 import { EthAddress } from "dcl-crypto";
 import { ContentAuthenticator } from "../auth/Authenticator";
-import { retry } from "@katalyst/content/helpers/RetryHelper";
+import { AccessCheckerForScenes } from "./AccessCheckerForScenes";
+import { AccessCheckerForProfiles } from "./AccessCheckerForProfiles";
+import { AccessCheckerForWearables } from "./AccessCheckerForWearables";
 
 export class AccessCheckerImpl implements AccessChecker {
 
     private static readonly LOGGER = log4js.getLogger('AccessCheckerImpl');
 
-    constructor(
-        private readonly authenticator: ContentAuthenticator,
-        private readonly fetcher: Fetcher,
-        private readonly dclParcelAccessUrl: string) { }
+    private readonly accessCheckerForScenes: AccessCheckerForScenes;
+    private readonly accessCheckerForProfiles: AccessCheckerForProfiles;
+    private readonly accessCheckerForWearables: AccessCheckerForWearables;
+
+    constructor(authenticator: ContentAuthenticator, fetcher: Fetcher, dclParcelAccessUrl: string) {
```

Comment:
Does the name `dclParcelAccessUrl` still make sense? Since we are still using it for wearables?

Reply:
In fact, it's a really good catch! It's a totally different URL!